less/mixins/text-emphasis.less

```diff
@@ -2,7 +2,7 @@
 
 .text-emphasis-variant(@color) {
   color: @color;
-  a&:hover {
+  a&:hover, a&:focus {
```

Comment:
I'm happy to fix that when rebasing this. Do you approve, in principle, of adding the selector?